app/Http/Client/Alchemy/AlchemyPendingRequest.php

```diff
@@ -619,21 +620,33 @@ private function extractTraits(array $nft): array
         /** @var array<array{trait_type: string | null, value: string | null, display_type: string | null}> $props */
         $props = Arr::get($nft, 'metadata.attributes', Arr::get($nft, 'metadata.properties', []));
 
-        return collect($props)
-            ->filter(function ($item) {
-                return ! empty(Arr::get($item, 'trait_type')) && ! empty(Arr::get($item, 'value'));
-            })
-            ->map(function ($item) {
-                $value = strval($item['value']);
-                $displayType = TraitDisplayType::fromAlchemyDisplayType(Arr::get($item, 'display_type'), $value);
-
-                return [
-                    'name' => $item['trait_type'],
-                    'value' => $value,
-                    'displayType' => $displayType,
-                ];
-            })
-            ->toArray();
+        try {
+            return collect($props)
+                ->filter(function ($item) {
+                    return ! empty(Arr::get($item, 'trait_type')) && ! empty(Arr::get($item, 'value')) && ! is_array(Arr::get($item, 'value'));
```

Comment:
here we currently ignore any arrays, not sure how to handle those properly at this time. Would need additional collections to get an idea of how they are used

Reply:
it seems opensea also ignores the array trait, see https://opensea.io/collection/sudolets (color is missing)